index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
- [ ] Background color of this section should be `#272A31. Based on project requirements it is necessary to use basic color swatch as provided in the theme: 
<img width="319" alt="Screenshot 2022-04-13 at 17 44 06" src="https://user-images.githubusercontent.com/77038610/163194199-3872a2ee-6ed1-432f-bc3e-57baef3e4c70.png">

- [ ] Please make sure to use same colors in the project. 
- [ ] You should also be using same fonts,  which are `Cocogoose, Lato`:
<img width="1045" alt="Screenshot 2022-04-13 at 17 45 41" src="https://user-images.githubusercontent.com/77038610/163194544-10e000d2-c657-4017-9205-5b11253cc1d8.png">

Reply:
hi @ShoiraTa 
Could you have a look on this piece of code I guess it is not the right one

Reply:
Hi @basem909 , can you tell more about the piece of code you are talking about ☀️

Reply:
Please, what section is it you wanted me to change its background color the code you quoted is at the head? 
@ShoiraTa

Reply:
hi @basem909 you had to change colors everywhere where you are using orange, dark blue and gray. I checked your code, all colors look the same good job 👍🏻 , only this part is remaining:
<img width="1393" alt="Screenshot 2022-04-15 at 08 50 21" src="https://user-images.githubusercontent.com/77038610/163519925-53c42b9c-7de3-4c58-8949-c70874a4f023.png">

Reply:
I just changed it from hex to rgba but the same the values. I am using the same exact colors you could check that the same values of the hex and rgba
thanks for clarification
@ShoiraTa

Reply:
Hi, the currently the rgba value is `rgba(39, 42, 49, 0.9)` its hex is `#272a31e6` this color is much lighter you can compare: 

`#272a31e6`  | #272A31
------------- | -------------
<img width="897" alt="Screenshot 2022-04-15 at 09 09 08" src="https://user-images.githubusercontent.com/77038610/163523472-b8dec66f-1f08-469f-9a79-e36e19043da1.png"> | <img width="803" alt="Screenshot 2022-04-15 at 09 09 55" src="https://user-images.githubusercontent.com/77038610/163523552-2ce11df0-0a79-4e54-9503-107f344bb111.png">

I understand you did it to make the melody logo visible, but even after making color brighter, the contrast is not good, I recommend using some other  lighter logo to make a better contrast and improve UI readability 👍🏻